content/millennium/r4/foundation/other/binary.md

```diff
@@ -9,11 +9,13 @@ title: Binary | R4 API
 
 ## Overview
 
-The Binary resource can contain any clinical content such as text, images, and PDFs. This resource is currently limited to Continuity of Care Documents (CCD), clinical documents, diagnostic reports, and communication message content. The supported diagnostic reports are Anatomic Pathology, Cardiology, Microbiology, and Radiology.
+The Binary resource can contain any clinical content such as text, images, and PDFs. This resource is currently limited to Continuity of Care Documents (CCD), clinical documents, diagnostic reports, and communication message content. 
+Example supported diagnostic report types include Anatomic Pathology, Cardiology, Microbiology, and Radiology.  
+_This list is not exhaustive and subject to change, as the list of clinical events that may qualify can be customized by clients_
 
-It is recommended to request Binary resources only after obtaining links to the resources via references from DiagnosticReport or DocumentReference. It is not recommended to start a workflow with Binary resources.
+It is recommended to request Binary resources only after obtaining links to the resources via references from [DiagnosticReport](http://fhir.cerner.com/millennium/r4/clinical/diagnostics/diagnostic-report) or [DocumentReference](http://fhir.cerner.com/millennium/r4/foundation/documents/document-reference). It is not recommended to start a workflow with Binary resources.
 
-The consumer must populate the Accept header with either `application/fhir+json` or the format returned in the `attachment.contentType` of the referring resource. If the Accept header is `application/fhir+json`, a FHIR Binary resource is returned with the raw data populated in `Binary.data`. If the Accept header is anything else, the raw data will be returned instead of a FHIR Binary resource. For more information, see the HL7<sup>®</sup> FHIR<sup>®</sup> [Binary documentation](http://hl7.org/fhir/r4/binary.html#rest).
+The consumer must populate the `Accept` header with either `application/fhir+json` or the format returned in the `attachment.contentType` of the referring resource. If the `Accept` header is `application/fhir+json`, a FHIR Binary resource is returned with the raw data populated in `Binary.data`. If the `Accept` header is anything else, the raw data will be returned instead of a FHIR Binary resource. For more information, see the HL7<sup>®</sup> FHIR<sup>®</sup> [Binary documentation](http://hl7.org/fhir/r4/binary.html#rest).
```

Comment:
\<sup> isn't necessary

Reply:
Didn't know, and didn't want to break it on accident, so I left it there.